common/djangoapps/student/management/commands/bulk_unenroll.py

```diff
@@ -22,20 +22,31 @@ class Command(BaseCommand):
     It expect that the data will be provided in a csv file format with
     first row being the header and columns will be either one of the
     following:
-    username,course-id
+    | username | course_id |
     OR
-    course-id
+    | course_id |
+
+    Example:
+            $ ... bulk_unenroll --csv_path=foo.csv
+            $ ... bulk_unenroll --csv_path=foo.csv --commit
```

Comment:
What is the reason we're using commit flag? And what will be the difference of using it or not using it?

Reply:
If you give `--commit` flag, the changes will be made, otherwise, the command can only dry run.